docs/docs/examples/storage/additional storage/ADDTIONAL_STORAGE.md

```diff
@@ -0,0 +1,13 @@
+# Additional Storage
+You can use [volumeClaimTemplates](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#volume-claim-templates) to have additional storage. This is useful when your enviroment uses several types of storage. 
+
+E.g. If you want to deploy Confluence on NFS, but you want to use BlockStorage (or everything else instead of NFS) for the lucene-index, you can create extra volumn for BlockStorage by defining volumeClaimTemplates in `values.yaml`.
+```yaml
+confluence:
```

Comment:
This should also show how to use the new volume with `additionalMounts` property.